src/main/resources/db/migration/common/V1_421__v72_2.sql

```diff
@@ -0,0 +1,111 @@
+ALTER TABLE courthouse
+    ADD COLUMN courthouse_object_id CHARACTER VARYING(16);
+ALTER TABLE courthouse
+    ADD COLUMN folder_path CHARACTER VARYING;
+
+-- add subcontent_object_id and subcontent_position to annotation_document
+ALTER TABLE annotation_document
+    ADD COLUMN subcontent_object_id CHARACTER VARYING(16);
+ALTER TABLE annotation_document
+    ADD COLUMN subcontent_position INTEGER;
+
+-- add subcontent_object_id and subcontent_position to daily_list
+ALTER TABLE daily_list
+    ALTER COLUMN content_object_id TYPE CHARACTER VARYING(16);
+ALTER TABLE daily_list
+    ADD COLUMN subcontent_object_id CHARACTER VARYING(16);
+ALTER TABLE daily_list
+    ADD COLUMN subcontent_position INTEGER;
+
+-- add subcontent_object_id and subcontent_position to media
+ALTER TABLE media
+    ADD COLUMN subcontent_object_id CHARACTER VARYING(16);
+ALTER TABLE media
+    ADD COLUMN subcontent_position INTEGER;
+
+-- add subcontent_object_id and subcontent_position to transcription_document
+ALTER TABLE transcription_document
+    ADD COLUMN subcontent_object_id CHARACTER VARYING(16);
+ALTER TABLE transcription_document
+    ADD COLUMN subcontent_position INTEGER;
+
+-- remove user_name from user_account
+UPDATE user_account
+SET user_full_name = user_name
+WHERE user_full_name IS NULL
+  AND user_name IS NOT NULL;
+ALTER TABLE user_account
+    DROP COLUMN user_name;
+
+-- amend user_account.user_full_name to not null
+ALTER TABLE user_account
+    ALTER COLUMN user_full_name SET NOT NULL;
+
+-- reintroduce user_login_name,user_os_name,user_login_domain,user_global_unique_id,user_ldap_dn to user_account
+ALTER TABLE user_account
+    ADD COLUMN user_os_name CHARACTER VARYING;
+ALTER TABLE user_account
+    ADD COLUMN user_ldap_dn CHARACTER VARYING;
+ALTER TABLE user_account
+    ADD COLUMN user_global_unique_id CHARACTER VARYING;
+ALTER TABLE user_account
+    ADD COLUMN user_login_name CHARACTER VARYING;
+ALTER TABLE user_account
+    ADD COLUMN user_login_domain CHARACTER VARYING;
+
+-- reinstate numeric user_state to user_account
+ALTER TABLE user_account
+    ADD COLUMN user_state SMALLINT;
+
+-- add extobjdir_process_detail as 1:1 with external_object_directory ( c.f.case_overflow )
+CREATE TABLE extobjdir_process_detail
+(
+    epd_id           INTEGER                  NOT NULL,
+    eod_id           INTEGER                  NOT NULL UNIQUE,
+    event_date_ts    TIMESTAMP WITH TIME ZONE,
+    update_retention BOOLEAN                  NOT NULL,
+    created_ts       TIMESTAMP WITH TIME ZONE NOT NULL,
+    created_by       INTEGER                  NOT NULL,
+    last_modified_ts TIMESTAMP WITH TIME ZONE NOT NULL,
+    last_modified_by INTEGER                  NOT NULL
+);
+
+ALTER TABLE extobjdir_process_detail
+    ADD CONSTRAINT epd_external_object_directory_fk
+        FOREIGN KEY (eod_id) REFERENCES external_object_directory (eod_id);
+
+-- TODO: FKs for created_by and last_modified_by
```

Comment:
Author note: Awaiting comment resolution on https://github.com/hmcts/darts-api/pull/2355.

Reply:
Resolving on the basis these FKs will be added in v72_3.